lib/PhpZendClientGenerator.php

```diff
@@ -432,7 +434,14 @@ function writeClass(DOMElement $classNode)
 					
 				case "string" :
 					$this->appendLine("		if(count(\$xml->{$propName}))");
-					$this->appendLine("			\$this->$propName = ($propType)\$xml->$propName;");
+					$multiLingualPropName = self::MULTILINGUAL_PREFIX . $propName;
+					$arrayType = $propertyNode->getAttribute ( "arrayType" );
+					$this->appendLine("		{");
+					$this->appendLine("			if(count(\$xml->{$propName}->item))");
```

Comment:
wouldn't that throw notice or warning if `$xml->{$propName}` is not an object? maybe add isset or is_object before?